translate/src/hooks/useLocaleRole.ts

```diff
@@ -0,0 +1,44 @@
+import { useContext } from 'react';
+
+import { Locale } from '~/context/Locale';
+import { USER } from '~/modules/user';
+import { useAppSelector } from '~/hooks';
+
+/**
+ * Return the user's status within the given locale, to display on the user banner
+ */
+export function useLocaleRole(): Array<string> {
+  const { code } = useContext(Locale);
+  const {
+    isAuthenticated,
+    isAdmin,
+    managerForLocales,
+    translatorForLocales,
+    dateJoined,
+  } = useAppSelector((state) => state[USER]);
+
+  if (!isAuthenticated) {
+    return ['', ''];
+  }
+
+  if (isAdmin) {
+    return ['ADMIN', 'Admin'];
+  }
+
+  if (Object.hasOwnProperty.call(managerForLocales, code)) {
```

Comment:
I believe this is clearer:

```suggestion
  if (managerForLocales.includes(code)) {
```